### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by war-in**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+$ universal-sierra-compiler \
+  --sierra-input-path ./path/to/sierra.json
+  --casm-output-path ./path/to/casm.json
+```
+
+> 📝 **Note**
+>
+> `casm.json` can be path to existing (or not) file.
+> If the file exists its contents will be overwritten.
+>
```
Comment (piotmag769) [inline README.md:32]: Don't think this is needed

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Usage
+
+To use the tool, just pass your Sierra json file path:
+
+```shell
+$ universal-sierra-compiler \
+  --sierra-input-path ./path/to/sierra.json
+  
+{"bytecode": ...}
+```
```
Comment (piotmag769) [inline README.md:14]: I don't see a use case for that, isn't it better to just enforce passing output path?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Universal-Sierra-Compiler
+
+Universal-Sierra-Compiler is the tool for Sierra compilation. It compiles any ever-existing Sierra version to the CASM.
```
Comment (piotmag769) [inline README.md:3]:
```suggestion
Universal-Sierra-Compiler is the tool for Sierra compilation. It compiles any ever-existing Sierra version to CASM.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+## Usage
+
+To use the tool, just pass your Sierra json file path:
```
Comment (piotmag769) [inline README.md:7]:
```suggestion
To use the tool, just pass a path to a file with Sierra in json format:
```

File: README.md (hunk 2 above)
Comment (war-in) [reply README.md:14]: Nope, it would be useful for us. We can read the compiled sierra from the stdout and bypass creating and removing the file.

**Review by Arcticae — CHANGES_REQUESTED** [review]
Let's add a table with supported sierra versions range so it's clear how to update, when we do it?

**Review by Arcticae — APPROVED** [review]

**Pull request merged**